cloud/pkg/cloudhub/servers/httpserver/signcerts.go

```diff
@@ -61,7 +61,9 @@ func getIps(advertiseAddress []string) (Ips []net.IP) {
 
 // GenerateToken will create a token consisting of caHash and jwt Token and save it to secret
 func GenerateToken() error {
-	expiresAt := time.Now().Add(time.Hour * 24).Unix()
+	// set double TokenRefreshDuration as expirationTime, which can assure validity period of
+	// token obtained anytime GE to TokenRefreshDuration
```

Comment:
okey, some minor changes for better understanding, ptal :)
```suggestion
	// set double TokenRefreshDuration as expirationTime, which can guaranteed that the validity period of the 
        // token obtained at anytime is greater than or equal to TokenRefreshDuration
```

Reply:
okay